### PR TITLE
signup confirmation by email with using rabbitmq added, some code cor…

### PR DESCRIPTION
**Pull request by groyvstreet**
…rections

### CODE REVIEW

File: Services/TokenAuthentication/ModsenOnlineStore.Login.Infrastructure/Services/LoginService.cs [hunk 1]
```diff
@@ -50,48 +54,99 @@ public async Task<DataResponseInfo<string>> GetToken(LoginData data)
...
+                                     arguments: null);
+
+                var message = user.Email + " " + user.Id + " " + emailConfirmation.Code;
+
+                var body = Encoding.UTF8.GetBytes(message);
+
+                channel.BasicPublish(exchange: "",
+                                     routingKey: "email-confirmation",
+                                     basicProperties: null,
+                                     body: body);
```
Comment (elv1sabene) [inline Services/TokenAuthentication/ModsenOnlineStore.Login.Infrastructure/Services/LoginService.cs]: move all stuff about messaging to another abstraction

**Review by elv1sabene — COMMENTED** [review]
add `Async` postfix to all methods returning `Task` or `Task<>`

File: Services/TokenAuthentication/ModsenOnlineStore.Login.Infrastructure/Services/LoginService.cs (hunk 1 above)
Comment (groyvstreet) [reply Services/TokenAuthentication/ModsenOnlineStore.Login.Infrastructure/Services/LoginService.cs]: fixed

File: Services/TokenAuthentication/ModsenOnlineStore.Login.Infrastructure/Services/LoginService.cs [hunk 2]
```diff
@@ -50,48 +58,90 @@ public async Task<DataResponseInfo<string>> GetToken(LoginData data)
...
+
+            var emailConfirmation = new EmailConfirmation
+            {
+                UserId = user!.Id,
+                Code = Guid.NewGuid().ToString()
+            };
+
+            await emailConfirmationRepository.AddEmailConfirmation(emailConfirmation);
+
+            var message = user.Email + " " + user.Id + " " + emailConfirmation.Code;
```
Comment (elv1sabene) [inline Services/TokenAuthentication/ModsenOnlineStore.Login.Infrastructure/Services/LoginService.cs]: use string interpolation

**Review by elv1sabene — APPROVED** [review]

**Pull request merged**